extension/benchmark/apple/Benchmark/Tests/Tests.xcconfig

```diff
@@ -16,7 +16,7 @@ OTHER_LDFLAGS[sdk=iphonesimulator*] = $(inherited) \
 -force_load $(BUILT_PRODUCTS_DIR)/libkernels_quantized-simulator-release.a
 
 HEADER_SEARCH_PATHS[sdk=iphoneos*] = $(inherited) \
-$(SRCROOT)/../../../.. \
+$(SRCROOT)/../../../../.. \
```

Comment:
@shoumikhin The relative path should be corrected right? Are there any other relative paths for this app?

Reply:
Yes, that's right